Source/Core/VideoBackends/D3D/Render.cpp

```diff
@@ -474,56 +474,54 @@ u32 Renderer::AccessEFB(EFBAccessType type, u32 x, u32 y, u32 poke_data)
 		else if (alpha_read_mode.ReadMode == 1) return (ret | 0xFF000000); // GX_READ_FF
 		else /*if(alpha_read_mode.ReadMode == 0)*/ return (ret & 0x00FFFFFF); // GX_READ_00
 	}
-	else if (type == POKE_COLOR)
+	else // if (type == POKE_COLOR || type == POKE_Z)
 	{
-		u32 rgbaColor = (poke_data & 0xFF00FF00) | ((poke_data >> 16) & 0xFF) | ((poke_data << 16) & 0xFF0000);
+		EfbPokeData d;
```

Comment:
I think this kind of fallback should be within the caller. So it could be removed from OGL too.